docs/docs.html

```diff
@@ -0,0 +1,765 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.4.0/font/bootstrap-icons.css">
+  <link rel="stylesheet" href="/css/docs.css" />
+  <link rel="stylesheet" href="/css/rabiui.components.css">
+  <title>Rabi UI | Docs</title>
+  <link rel="shortcut icon" href="/images/favicon.ico" type="image/x-icon">
+</head>
+
+<body>
+  <div class="grid-row">
+    <nav class="navbar fix-top">
+      <div class="nav-left">
+        <div class="nav-brand">
+          <h1>
+            <a class="brand-logo" href="/index.html"><span class="nav-highlight">Rabi</span> UI</a>
+          </h1>
+        </div>
+        <div class="version-mobile">
```

Comment:
I don't think you need a different HTML Structure for mobile and desktop view. You can play with media queries and classes. It's okay for now. Just remember it for future projects.

Reply:
Actually, the position of that div changes w.r.t. screen size. So I had to do it like this.

Reply:
Yeah, you can change the position of div with media queries.